toml/src/test/kotlin/org/rust/toml/CargoTomlDependenciesCompletionTest.kt

```diff
@@ -165,6 +165,16 @@ class CargoTomlDependenciesCompletionTest : CargoTomlCompletionTestBase() {
         }
     }
 
+    private fun doCompletionTest(
+        @Language("TOML") before: String,
+        @Language("TOML") after: String,
+        crate: Pair<String, String>
+    ) {
+        withMockedFullCrateDescription(CrateFullDescription(crate.first, crate.second, emptyList())) {
```

Comment:
> Why emptyList()?

Because the current production code leverages `maxVersion` instead of the versions list. I have however replaced this with a proper version now.